library/general/src/modules/Mode.rb

```diff
@@ -293,6 +293,12 @@ def config
       mode == "autoinst_config"
     end
 
+    # AutoYaST is running in the clone_system mode
+    # which is called by "yast clone_system"
+    def autoyast_clone_system
+      ARGV[0] == "clone_system"
```

Comment:
what about cloning in AY UI? there is also clone button and also menu option to clone system. So it is not called just in this one case. Maybe it will be better to set some flag in country when auto with change is called? So we know user set it in UI? https://www.rubydoc.info/github/yast/yast-yast2/Installation/AutoClient#change-instance_method

Reply:
Hm, I am expecting these problems in other modules too (e.g. I have seen it in the ssh export too ) So I would prefer a global place for that decision in Mode module.
If it is needed we also could set the Mode.clone_mode in the AY UI too, but up to now I do not see a need for it.
So I think we are more flexible the the "Mode solution".
Other opinion ?

Reply:
what is actually goal? you need to use mode in every module or do you want to use mode in AY only when cloning? if later, then you can remember in AY for which modules UI is ran and base it on it?